src/reducers/fiatToCryptoReducer.js

```diff
@@ -18,34 +18,89 @@
     51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 */
 
-import { SET_ALTALIX_INFO } from 'constants/fiatToCryptoConstants';
+import {
+  SET_ALTALIX_INFO,
+  SET_SENDWYRE_RATES,
+  LOAD_SENDWYRE_COUNTRY_SUPPORT,
+  SET_SENDWYRE_COUNTRY_SUPPORT,
+  RESET_SENDWYRE_COUNTRY_SUPPORT,
+  SENDWYRE_SUPPORT,
+} from 'constants/fiatToCryptoConstants';
 
-type AltalixInfo = {
-  isAvailable: boolean,
-};
+import type { SendwyreRates } from 'models/FiatToCryptoProviders';
 
-export type FiatToCryptoReducerState = {
-  altalix: null | AltalixInfo,
-};
+export type FiatToCryptoReducerState = {|
+  isAltalixAvailable: null | boolean,
+  sendwyreExchangeRates: null | SendwyreRates,
```

Comment:
```suggestion
  isAltalixAvailable: ?boolean,
  sendwyreExchangeRates: ?SendwyreRates,
```

Reply:
I'd prefer to have only one value that means "no data" to compare against - if I understand correctly, `?type` would also accept `undefined`.

Reply:
Yes, `undefined` would also be accepted. In that case, leave it as it is